subsys/nfc/tnep/tag.ld

```diff
@@ -0,0 +1 @@
+Z_ITERABLE_SECTION_ROM(nfc_tnep_tag_service, 4)
```

Comment:
What does 4 mean here? Does it stand for the number of section items that you can add (max 4 TNEP services)?

If so, please check if KConfig symbols are visible here and define this value as a parameter